src/Storages/S3Queue/S3QueueFilesMetadata.cpp

```diff
@@ -363,7 +362,7 @@ std::pair<S3QueueFilesMetadata::SetFileProcessingResult,
 }
 
 std::pair<S3QueueFilesMetadata::SetFileProcessingResult,
-          S3QueueFilesMetadata::ProcessingNodeHolderPtr> S3QueueFilesMetadata::trySetFileAsProcessingForOrderedMode(const std::string & path)
```

Comment:
Minor. Might be better to pass it around using `const FileStatusPtr &` to avoid unnecessary atomic ops on refcounter.